### PR TITLE
V1.1.0

### PR DESCRIPTION
**Pull request by DougManuel**
Addresses: #59, #84, #90, #92, #93, #96, #97, #99, #102, #103, #104, #113, #114, #116. 

Also see changelog.

Before merging to main:
- @hswerdfe volunteered to clean-up merge conflict with CSV.
- update changelog.
- check ERD and SQL reflects model.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,139 +24,213 @@ See [license](LICENSE) for more information.
...
+- Reporter
+  - `organization`: Organization of reporter. Issue [#97](https://github.com/Big-Life-Lab/covid-19-wastewater/issues/97)
+
+- Sample
+  - `reporterID`: Reporter ID. Currently, reporterID is `WWmeasure` table but reporter for samples can be different. Issue [#93](https://github.com/Big-Life-Lab/covid-19-wastewater/issues/93)
+  - `index`: Index number in case the sample was taken multiple times. Issue [#103](https://github.com/Big-Life-Lab/covid-19-wastewater/issues/103) 
+  - `InstrumentID`: Links with the Instrument table to describe the instrument used for sampling. Issue [#104](https://github.com/Big-Life-Lab/covid-19-wastewater/issues/104) 
+
+- SiteMeasure
+  - `SampleID`: Makes sure that samples can easily be linked back to the site measurements, without the need for comparing dates. In case that multiple samples need to be linked to the same site measurement, create a comma separated list of sample IDs.
```
Comment (vipileggi) [inline README.md:53]: should this not be `sampleID`

File: README.md [hunk 2]
```diff
@@ -24,139 +24,213 @@ See [license](LICENSE) for more information.
...
+
+  - Categories added to allow variant reporting. See `WWMeasure` table, `type` variable:
+
+    - `varB117`: Variant B.1.1.7
+    - `varB1351`: Variant B.1.351
+    - `varP1`: Variant P.1
+
+  - Updated description of `WWMeasure` table, `unit` measure. These descriptions now reference gene or variant copies.
+  - New `detected`: Gene copies or variant detected in the sampleGene or variant copies. Detected = 1. Gene or variant copiesNot detected = 0.
+  - New `porpVar`: Proportion of variant in sample.
```
Comment (vipileggi) [inline README.md:71]: may want to change to `propVar`  from `porpVar`

File: README.md [hunk 3]
```diff
@@ -24,139 +24,213 @@ See [license](LICENSE) for more information.
...
+  - `SiteMeasure` table, `type` variable
+    - `wwBOD5c`, 5 day biochemical oxygen demand
```
Comment (vipileggi) [inline README.md:73]: if this is total BOD5 may want to change to: `wwBOD5t, total 5 day...`; if its intended to be carbonaceous then add, ' 5 day carbonaceous biochemical...' to distinguish from nitrogenous BOD.

File: README.md [hunk 4]
```diff
@@ -24,139 +24,213 @@ See [license](LICENSE) for more information.
...
+    - `wwPtot`, Total phosphates
```
Comment (vipileggi) [inline README.md:74]: MECP lab typically measures total phosphorous (TP) `wwTP`; but this is fine also but not too common from my experience.

File: README.md [hunk 5]
```diff
@@ -24,139 +24,213 @@ See [license](LICENSE) for more information.
...
+  - New `dailyAvg`: Average value taken over a 24h period, in the `SiteMeasure` table, `aggregation` variable
+  - The `SiteMeasure` table now has a categorical `unit` variable with the following options
+
+    - `°C`, Degrees Celcius
+    - `mm`, Millimeters
+    - `m3/h`, Cubic meters an hour
```
Comment (vipileggi) [inline README.md:80]: `... per hour`

File: README.md [hunk 6]
```diff
@@ -24,139 +24,213 @@ See [license](LICENSE) for more information.
...
+    - `m3/d`, Cubic meters a day
```
Comment (vipileggi) [inline README.md:81]: `... per day`

File: README.md [hunk 7]
```diff
@@ -24,139 +24,213 @@ See [license](LICENSE) for more information.
...
-    -   `typeOther`: (NEW) Description of the instrument in case it is not listed in instrumentType.
-    -   `aggregation`: (NEW) When reporting an aggregate measurement, this field describes the method used.
-    -   `aggregationOther`: (NEW) Description for other type of aggregation not listed in aggregation.
-    -   `aggregationDescription`: (NEW) Information on OR reference to which measurements that were included to calculate the aggregated measurement that is being reported.
-    -   `value`: (NEW) The actual value that is being reported for this measurement.
-    -   `unit`: (NEW) The engineering unit of the measurement.
-    -   `qualityFlag`: (NEW) Does the reporter suspect quality issues with the value of this measurement? (Boolean)
-    -   `notes`: (NEW) Any additional notes.
+  - `ID`: (NEW) Unique identifier for each contextual measurement.
+  - `SiteID`: (NEW) Links with the Site table to describe the location of measurement.
```
Comment (vipileggi) [inline README.md:125]: `siteID` ?

File: README.md (hunk 3 above)
Comment (hswerdfe) [reply README.md:73]: to discuss I think

File: README.md (hunk 4 above)
Comment (hswerdfe) [reply README.md:74]: to discuss

File: README.md (hunk 5 above)
Comment (hswerdfe) [reply README.md:80]: This listed change in the DM does not seem to actually be in the DM.
 * The `SiteMeasure` table now has a categorical `unit` variable with the following options
units is still listed as string in `dev` branch

**Comment by DougManuel** [comment]
There are a few open issues. I am not sure if they just need to be closed or still work to do.

#101, #106, #112, #113.

**Review by hswerdfe — APPROVED** [review]

**Pull request merged**